src/charts/loading/LoadingContainer.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+
+import LoadingContainer from './LoadingContainer';
+
+import barData from '../bar/barChart.fixtures';
+import bar from '../bar/barChart';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Loading Container', () => {
+
+    it('should render without errors', () => {
+        expect(() => (
+            mount(
+                <LoadingContainer
+                    data={barData}
+                    loadingState={'.load-state'}
+                >
+                    <div className="chart" />
+                </LoadingContainer>
+            )
+        ).unmount()).not.toThrow();
```

Comment:
I was surprised by this 'unmount'

Reply:
We use this pattern in one of our tests to ensure that we unmount the component after the test.

However, I don't think we need to worry about unmounting everything in britecharts-react, that restriction is only for `core-js` tests.